edudoc/judge-tutorial/judge-tutorial-pt.md

```diff
@@ -1,77 +1,80 @@
 # Tutorial de Ajuizamento para Iniciantes
 
-### Version 1.0 {.version}
+### Versão 2.1 {.version}
 
 Este documento destina-se àqueles que nunca foram juízes e destacará alguns dos principais regulamentos a serem seguidos durante o processo de julgamento. O procedimento descrito destina-se a eventos regulares de resolução em velocidade. Para eventos de os olhos vendados, existem algumas diferenças importantes não destacadas neste documento.
 
 ## Equipamento
 
 Este é o principal equipamento e ferramentas necessárias para realizar uma tentativa de resolução em velocidade:
 
-**Cronómetro Stackmat**: Este é o dispositivo oficial para medir tempos numa competição WCA. A imagem abaixo é a versão mais recente do cronómetro, mas as versões 4 e 3 do cronómetro são também aceites em competições.
+**Cronómetro Stackmat**: Este é o dispositivo oficial para medir tempos numa competição WCA. A imagem abaixo é a versão mais recente do cronómetro (Gen 5), as versões Gen 3 e Gen 4 do cronómetro são também aceites em competições.
 
-![](images/timer-pt.png){.centered width=65%}
+![](images/timer-pt.png){.centered}
```

Comment:
Why did this get changed? It appears that this file doesn't match the English version. Can you please go through this translation and the English version side-by-side and make sure the formatting is the same?